distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/InstallPluginCommandTests.java

```diff
@@ -430,6 +431,13 @@ public void testMalformedUrlNotMaven() throws Exception {
         assertTrue(e.getMessage(), e.getMessage().contains("no protocol"));
     }
 
+    public void testFileNotMaven() throws Exception {
+        Tuple<Path, Environment> env = createEnv(fs, temp);
+        // has two colons, so it appears similar to maven coordinates
+        FileNotFoundException e = expectThrows(FileNotFoundException.class, () -> installPlugin("file:C:\\plugin.zip", env.v1()));
+        assertTrue(e.getMessage(), e.getMessage().startsWith("C:\\plugin.zip"));
+    }
```

Comment:
Can you please use a  filename that is a lot more explicit and unlikely to "accidentally" exist if on a windows box? And please run this test on windows (I'm a little worried we won't always get FileNotFoundException, for example getting an AccessDeniedException on some systems instead).

Reply:
Thanks, I randomized the directory name to test and changed the assertion to check the message of the exception only (I tested on windows server and got a FileNotFoundException consistently so I can put it back if you want):
https://github.com/elastic/elasticsearch/pull/28163/commits/0779d60a87f1f34da06c94ea470e5ca28ace20b2